reference/tax_settings.v3.yml

```
@@ -143,41 +143,22 @@ components:
-    Tax_Settings_Req:
-      type: object
-      properties:
-        tax_entered_with_prices:
-          type: boolean
-          description: Whether prices entered on this store include a tax component or not.
-        price_display_settings:
-          type: object
-          description: Settings that describe how prices display at the global level.
-          properties:
-            show_inclusive_in_control_panel:
-              type: boolean
-              description: Whether to show prices as tax inclusive or tax exclusive in the BigCommerce control panel.
-            invoice_price_display_strategy:
-              type: string
-              description: 'Whether to show prices as tax inclusive or tax exclusive across all invoices, or use the shopperʼs tax zone for price display on invoices.'
-              enum:
-                - ZONE
-                - INCLUSIVE
-                - EXCLUSIVE
-        fallback_strategy:
-          type: string
-          description: 'Decribes the fallback behaviour that applies when a tax provider produces an error. A merchant may decide to use a flat 10% fallback tax rate, their basic tax settings, or to block the transaction until a successful result can be achieved.'
-          enum:
-            - FIXED
-            - BASIC
-            - DISABLE
         should_subtract_store_tax:
           default: true
           type: boolean
           description: This setting applies only if a merchant enters tax-inclusive prices. When enabled, the store subtracts the itemʼs store tax rate before calculating tax using the shopperʼs tax zone. The tax-exclusive amount will be the same across all tax zones. When disabled, the tax-inclusive price remains the same across all tax zones; only the tax amount will vary based on the shopperʼs location. The tax-exclusive amount may vary among tax zones. These calculations are relevant for tax pricing and tax quotations that use basic tax.
+        should_use_geolocation_to_determine_guest_shopper_tax_zone:
+          default: false
+          type: boolean
+          description: This setting determines which tax zone a store uses to estimate tax for guest shoppers. When enabled, the store identifies a country-level tax zone based on the geolocation of a guest shopper. The store then applies the corresponding tax zone to estimate taxes. When disabled, the store identifies the zone using the provided `guest_shopper_tax_zone_id` field instead. Only the tax zones you configure can be matched to the guest shopper's geolocation.
+        guest_shopper_tax_zone_id:
+          default: 1
+          type: integer
+          description: ID for the tax zone a store uses when estimating tax for guest shoppers. The store uses this zone if you disable `should_use_geolocation_to_determine_guest_shopper_tax_zone`. The store also uses this zone if there is no matching country-level tax zone for the geolocation.
```

```suggestion
          description: ID for the tax zone that the store is set to use when estimating tax for guest shoppers. The geo-location that the store relies on when `should_use_geolocation_to_determine_guest_shopper_tax_zone` is enabled. The default for any store relies on this zone if there is no matching country-level tax zone for the geolocation.
```
You don't have to use this verbatim, just trying to rearrange some of the words to make it less like the store performs things like a person.

---

I might leave it because the meaning changes and it's less concise